### PR TITLE
Allow UniformGrid to have auto-width columns

### PR DESCRIPTION
**Pull request by DamienSAVOYAT**

### CODE REVIEW

**Comment by mathieumack** [comment]
@DamienSAVOYAT Please sync your branch with master.
Solution has been updated to be more compliant with yaml configuration and builds (separate pipelines and sources)

**Comment by DamienSAVOYAT** [comment]
> @DamienSAVOYAT Please sync your branch with master.
> Solution has been updated to be more compliant with yaml configuration and builds (separate pipelines and sources)



> @DamienSAVOYAT Please sync your branch with master.
> Solution has been updated to be more compliant with yaml configuration and builds (separate pipelines and sources)

Ok, it's done.

File: src/OpenXMLSDK.Engine/Word/ReportEngine/Models/UniformGrid.cs [hunk 1]
```diff
@@ -15,6 +15,11 @@ public class UniformGrid : Table
         /// </summary>
         public bool CantSplitRows { get; set; }
 
+        /// <summary>
+        /// Indicate the number of column for spliting the grid 
+        /// </summary>
+        public string ColumnNumber { get; set; }
```
Comment (mathieumack) [inline src/OpenXMLSDK.Engine/Word/ReportEngine/Models/UniformGrid.cs:21]: string value for a number ?
Please use a nullable int instead

**Review by mathieumack — CHANGES_REQUESTED** [review]

**Comment by mathieumack** [comment]
Hi, please fix comments in order to validate the PR.

File: src/OpenXMLSDK.Engine/Word/ReportEngine/Models/UniformGrid.cs (hunk 1 above)
Comment (DamienSAVOYAT) [reply src/OpenXMLSDK.Engine/Word/ReportEngine/Models/UniformGrid.cs]: I replace ColumnNumber by ColumnNumberKey because it's a key coming from context.

**Comment by DamienSAVOYAT** [comment]
Ok, I update code and currently work on this PR.

**Review by MarieAgnesLopez — APPROVED** [review]

**Review by Alteoris — APPROVED** [review]

**Review by HisCodeness — APPROVED** [review]

**Pull request merged**